src/leanback/java/com/archos/mediacenter/video/leanback/presenter/ListPresenter.java

```diff
@@ -53,7 +54,7 @@
         public ListViewHolder(Context context) {
             super(new BaseCardView(context));
             mBaseCardView = (BaseCardView)view;
-            mBaseCardView.setBackgroundColor(context.getResources().getColor(R.color.lb_basic_card_bg_color));
```

Comment:
Why do you need to specify the rgb value and not rely on lb_basic_card_bg_color ?